tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -1500,6 +1500,18 @@
             "url": "/_mozilla/css/float_intrinsic_width_a.html"
           }
         ],
+        "css/float_kerning_a.html": [
+          {
+            "path": "css/float_kerning_a.html",
+            "references": [
+              [
+                "/_mozilla/css/float_kerning_ref.html",
+                "=="
+              ]
+            ],
+            "url": "/_mozilla/css/float_kerning_a.html"
+          }
+        ],
```

Comment:
There's a `.items.reftest` section and a `.reftest_nodes` section in `MANIFEST.json`. I'm not sure the pattern for where the test definition should go, so I put it in `.items.reftest`, and it's properly picked up by `./mach test-wpt`, so I think it's happy